config.js

```diff
@@ -38,6 +38,8 @@ const DEFAULT_DB_CONNECTION_POOL = { // sequelize defaults
 };
 const hiddenRoutes = pe.HIDDEN_ROUTES ?
   pe.HIDDEN_ROUTES.split(',') : ['']; // Routes to hide
+const corsRoutes = pe.CORS_ROUTES ?
```

Comment:
document the feature and flags in a new section on the https://salesforce.github.io/refocus/docs/10-security.html page